libs/cardano-ledger-conformance/src/Test/Cardano/Ledger/Conformance/ExecSpecRule/Conway/Base.hs

```diff
@@ -267,12 +268,8 @@ ratifyEnvSpec ConwayRatifyExecContext {crecGovActionMap} =
         exists
           ( \eval ->
               pure $
-                Set.map
-                  CommitteeHotCredential
-                  ( Set.intersection
-                      (authorizedHotCommitteeCredentials (eval committeeState))
-                      ccVotes
-                  )
+                Set.map CommitteeHotCredential ccVotes
+                  `Set.intersection` foldr' Set.insert mempty (eval cs)
```

Comment:
Out of curiosity, why this change?

Reply:
Basically this function should compute `common` when given the other sets. It was completely wrong before, but it only matters when we want to check if some value conforms to the constraint so it's not awfully important for conformance testing where we only care about generating values.